ui/tests/acceptance/pki/pki-engine-workflow-test.js

```diff
@@ -307,11 +307,15 @@ module('Acceptance | pki workflow', function (hooks) {
       await visit(`/vault/secrets/${this.mountPath}/pki/keys`);
       await click(PKI_KEYS.generateKey);
       assert.strictEqual(currentURL(), `/vault/secrets/${this.mountPath}/pki/keys/create`);
-      await fillIn(GENERAL.inputByAttr('type'), 'exported');
+      await fillIn(GENERAL.inputByAttr('type'), 'exported'); // exported keys generated private_key data
       await fillIn(GENERAL.inputByAttr('keyType'), 'rsa');
       await click(GENERAL.saveButton);
       keyId = find(GENERAL.infoRowValue('Key ID')).textContent?.trim();
-      assert.strictEqual(currentURL(), `/vault/secrets/${this.mountPath}/pki/keys/${keyId}/details`);
+      assert.strictEqual(
```

Comment:
see comment [here](https://github.com/hashicorp/vault/pull/28705/files#diff-cdd194c55cf29d876d7820ed30e0fda497339127eaf95753e144c52626006185R61)

After upgrading to `5.3.2` Ember data no longer "stores" data returned from a save response to the record. Sometimes performing a save operation returns generated data (like `private_key` info here) 

Previously, any data returned after saving a record was stored in that model in the ember-data store until the page was hard refreshed (or the user navigated away from the route). It did not matter that we re-requested the record fro the server, any server data was just added to the record.

Now, this data is no longer cached so when we transition to details, data like generated `private_key` is lost because re-requesting the record from the server clears out the record and only data returned by the server is available.

It's important that we return generated data from saving a record, so the user can download or copy it. So we no longer transition and instead show the details on the same page.

![image](https://github.com/user-attachments/assets/e12c350e-7c98-4b89-a0f0-001f43f0dd61)